src/sentry/tasks/reports.py

```diff
@@ -817,11 +821,27 @@ def to_calendar(interval, series):
 
     series_value_map = dict(series)
 
+    # If global views are enabled we can generate a link to the day
+    has_global_views = features.has("organizations:global-views", organization)
+
     def get_data_for_date(date):
         dt = datetime(date.year, date.month, date.day, tzinfo=pytz.utc)
         ts = to_timestamp(dt)
         value = series_value_map.get(ts, None)
-        return (dt, {"value": value, "color": value_color_map[value]})
+
+        data = {"value": value, "color": value_color_map[value], "url": None}
+        if has_global_views:
+            url = f"/organizations/{organization.slug}/issues/"
```

Comment:
I think we should probably use `reverse ` here rather than hardcoding the url.

Reply:
Hm, I was planning to use `reverse`, but the code that essentially does this in the `Project` model doesn't, so I more or less just used that https://github.com/getsentry/sentry/blob/7d20c6e1f809e40d6ccbf04b1f0ea3a4b948ada2/src/sentry/models/project.py#L149-L155

Reply:
I wonder if there's a reason

Reply:
Looks like it was done here, although I don't understand why: https://github.com/getsentry/sentry/commit/d10e0d5187622aab6b6051882a52ec065dd81b42

Reply:
I'd say don't follow this pattern unless we can see an obvious reason

Reply:
Fixed.